README.md

```diff
@@ -28,29 +28,49 @@ Functions to perform should be stored in the following json format:
 {
   "ensemble_list": [
     {
-      "title": "example1",
-      "function": somefunc,
-      "inputs": [],
-      "start_time": {
-          "hour": 12,
-          "minute": 0,
-          "second": 0
-      },
-      "iterations": 5,
-      "interval": 60
-    },
-    {
-      "title": "example2",
-      "function": somefunc,
-      "inputs": [],
-      "start_time": {
-          "hour": 12,
-          "minute": 0,
-          "second": 0
-      },
-      "iterations": 5,
-      "interval": 60
-    }
+            "title": "zeta2",
+            "function": "ensemble_tests.tester_functions:print_this",
+            "inputs": [
+                "Hello World!"
+            ],
+            "start_time": {
+                "hour": 8,
+                "minute": 0,
+                "second": 30
+            },
+            "iterations": 5,
+            "interval": 60
+        },
+        {
+            "title": "dummy1",
+            "function": "ensemble_tests.tester_functions:add_this",
+            "inputs": [
+                1,
+                2
+            ],
+            "start_time": {
+                "hour": 12,
+                "minute": 0,
+                "second": 0
+            },
+            "iterations": 5,
+            "interval": 60
+        },
+        {
+            "title": "alpha1",
+            "function": "ensemble_tests.tester_functions:subtract_this",
+            "inputs": [
+                1,
+                2
+            ],
+            "start_time": {
+                "hour": 12,
+                "minute": 0,
+                "second": 0
+            },
+            "iterations": 5,
+            "interval": 60
+        }
     ...etc...
   ]
 }
```

Comment:
Specifications below this block should illustrate the new requirements. For example, `function: path to a callable function followed by function name, separated by a dot` and `inputs: list of arguments to the given function, which the user must confirm are the correct type`